### PR TITLE
add SL template for install failed due to iam:CreateServiceLinkedRole per…

### PR DESCRIPTION
**Pull request by bmeng**
…mission issue

### CODE REVIEW

File: osd/aws/InstallFailed_CreateServiceLinkedRole.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "severity": "Error",
+  "service_name": "SREManualAction",
+  "summary": "Installation blocked, action required",
+  "description": "Your cluster's installation was blocked by the ELB Service Role does not exist. In AWS accounts that have never created a load balancer before, it’s possible that the service role for ELB might not exist yet. Please refer to the troubleshooting doc: 'https://docs.openshift.com/rosa/rosa_support/rosa-troubleshooting-deployments.html#rosa-troubleshooting-elb-serivce-role_rosa-troubleshooting-cluster-deployments'",
```
Comment (cattias) [inline osd/aws/InstallFailed_CreateServiceLinkedRole.json:5]: Your cluster's installation was blocked **because** the ELB Service Role does not exist

Comment (bmeng) [reply osd/aws/InstallFailed_CreateServiceLinkedRole.json]: thanks! fixed

**Pull request merged**